flow/scripts/open.tcl

```diff
@@ -41,7 +41,11 @@ proc read_timing {input_file} {
   if {$design_stage >= 6 && [file exist $::env(RESULTS_DIR)/6_final.spef]} {
     log_cmd read_spef $::env(RESULTS_DIR)/6_final.spef
   } elseif {$design_stage >= 5} {
-    log_cmd estimate_parasitics -global_routing
+    if { [grt::have_routes] } {
+      log_cmd estimate_parasitics -global_routing
+    } else {
+      puts "No global routing results: load $::global_route_congestion_report for details"
```

Comment:
If GUI=1 then give a warning about missing parasitics for timing.

Reply:
Added warning, but why should GUI=1/0 be different in open/gui?

Shouldn't the only difference between open/gui_ be console vs. GUI?

Reply:
Sorry I meant GUI_TIMING=1

Reply:
Yes, GUI_TIMING=1/0 should have the same behavior. GUI vs. console; same behavior here.